src/components/gltf-model-plus.js

```diff
@@ -656,7 +664,55 @@ export async function loadGLTF(src, contentType, onProgress, jsonPreprocessor) {
     .register(parser => new GLTFHubsPlugin(parser, jsonPreprocessor))
     .register(parser => new GLTFHubsLightMapExtension(parser))
     .register(parser => new GLTFHubsTextureBasisExtension(parser))
-    .register(parser => new GLTFMozTextureRGBE(parser, new RGBELoader().setDataType(THREE.HalfFloatType)));
+    .register(parser => new GLTFMozTextureRGBE(parser, new RGBELoader().setDataType(THREE.HalfFloatType)))
+    .register(parser => new GLTFLodExtension(parser, {
+      loadingMode: 'progressive',
+      onLoadMesh: (lod, mesh, level, lowestLevel) => {
+        // Higher levels are progressively loaded on demand.
+        // So some post-loading processings done in gltf-model-plus and media-loader
+        // need to be done here now.
+
+        // Nothing to do if this is the lowest level mesh.
+        if (level === lowestLevel || lod.levels.length === 0) {
+          return mesh;
+        }
+
+        let lowestMeshLevel = null;
+        for (let index = lowestLevel; index > level; index--) {
+          if (lod.levels[index].object.type !== 'Object3D') {
+            lowestMeshLevel = index;
+            break;
+          }
+        }
+
+        if (lowestMeshLevel === null) {
+          return mesh;
+        }
+
+        // Create a mesh clone. Otherwise if an lod instance is cloned before higher
+        // levels are loaded the lods instance can refer to the same mesh instance,
+        // therefore the lods can be broken because an object can't be placed
+        // at multiple places in a Three.js scene tree.
+        mesh = mesh.clone();
+
+        convertStandardMaterialsIfNeeded(mesh);
+
+        // A hacky solution. media-loader and media-utils make a material clone
```

Comment:
Yeah I really don't lik ehow we are patching those materials. It has lead to lots of confusing bugs where things hold onto stale references of the pre-patched materials. We should think about this as part of new gltf loading stuff.